util/translate_timeline.js

```diff
@@ -87,16 +92,40 @@ const run = async (args) => {
     let line = timelineLines[i].trim();
 
     const lineText = lineToText[lineNumber];
-    if (lineText)
-      line = line.replace(` "${lineText.name}"`, ` "${lineText.text}"`);
-    const lineSync = lineToSync[lineNumber];
-    if (lineSync)
-      line = line.replace(`sync /${lineSync.origRegexStr}/`, `sync /${lineSync.regex.source}/`);
-
-    if (syncErrors[lineNumber])
-      line += ' #MISSINGSYNC';
-    if (textErrors[lineNumber])
-      line += ' #MISSINGTEXT';
+
+    if (args?.colorize === 'true') {
+      if (lineText)
+        line = line.replace(` "${lineText.name}"`, ` \x1b[93m"${lineText.text}"\x1b[0m`);
+      const lineSync = lineToSync[lineNumber];
+      if (lineSync)
+        line = line.replace(`sync /${lineSync.origRegexStr}/`, `\x1b[31msync\x1b[0m \x1b[32m/${lineSync.regex.source}/\x1b[0m`);
```

Comment:
This whole `(args?.colorize)` block feels like a repeat of the below.

What about adding a `colorize()` function? e.g.
```
const colorize = (str, colorStr) => {
  if (args?.colorize !== 'true')
    return str;
  // do some lookup of color names here,
  const colorCode = /* something with colorStr */
  return `${colorCode}${str}${endColorCode}`;
};

// ... later ...

      if (lineText)
        line = line.replace(` "${lineText.name}"`, ` "${colorize(lineText.text, 'yellow')}"`);
```